src/crimson/osd/pg_meta.cc

```diff
@@ -53,7 +54,11 @@ seastar::future<epoch_t> PGMeta::get_epoch()
         assert(epoch);
         return seastar::make_ready_future<epoch_t>(*epoch);
       }
-    });
+    }).handle_error(
+      crimson::ct_error::all_same_way([] {
```

Comment:
The alternative, perhaps more explicit-but-labor-requiring variant is `ll_read_errorator::all_same_way`. It was intended to allow handling a subset of errors IIRC. I'm curious which one prevail as a usual convention.

Reply:
it'd go with `FuturizedStore::read_errorator::assert_all`.